src/analyze.ts

```diff
@@ -780,13 +789,14 @@ module.exports = async function (id, code, job) {
   }
 
   function emitStaticChildAsset () {
```

Comment:
Should `emitStaticChildAsset()` accept any parameters? I see a couple usages like `emitStaticChildAsset(staticBindingsInstance)` and `emitStaticChildAsset(path)` that seem to indicate it should accept a parameter?

Reply:
This is left over from ncc here - https://github.com/vercel/webpack-asset-relocator-loader/blob/master/src/asset-relocator.js#L1210.

These paths can all be removed as we don't need to manipulate output in this project so yes any argument computation code paths can be deleted as well.

Reply:
Ok, I updated the calls to `emitStaticChildAsset()` so they no longer use any parameters.